chat-ui/src/services/conversations.test.ts

```diff
@@ -0,0 +1,261 @@
+import { vi } from "vitest";
+import { ConversationService, formatReferences } from "./conversations";
+import { References } from "chat-core";
+// import { MockEvent, EventSource } from "mocksse";
+import * as FetchEventSource from "@microsoft/fetch-event-source";
+
+/**
+ * Mocks
+ */
+
+/* Mock fetch for regular awaited HTTP requests */
```

Comment:
Prefer line comments `//` - block comments are annoying when you're working and want to block comment something out
```suggestion
// Mock fetch for regular awaited HTTP requests
```